src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -494,8 +596,20 @@ def continuing_criterion(carry: OptimizationCarry) -> jtp.Bool:
             has_aux=True,
         )
 
-        # Reshape the optimized solution to be a matrix of 3D contact forces.
-        CW_fl_C = solution.reshape(-1, 3)
+        if n_kin_constraints > 0:
+            # Extract the last n_kin_constr values from the solution and split them into 6D wrenches
+            kin_constr_wrench_inertial = solution[-n_kin_constraints:].reshape(-1, 6)
+
+            # Form an array of tuples with each wrench and its opposite using jax constructs
+            kin_constr_wrench_pairs_inertial = jax.vmap(
+                lambda wrench: jnp.array((wrench, -wrench))
+            )(kin_constr_wrench_inertial)
```

Comment:
Have you tried this?
```suggestion
            kin_constr_wrench_pairs_inertial = jnp.stack(
                (kin_constr_wrench_inertial, -kin_constr_wrench_inertial), axis=1
            )
```

Reply:
Thanks! Done in 23ea9ab6a15a85552e36d9d7863a4fbdc753eb97